docker/e2e/Dockerfile

```diff
@@ -27,6 +27,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 RUN wget https://raw.githubusercontent.com/google/timesketch/master/requirements.txt
 RUN pip3 install -r requirements.txt
 
+# TODO: Remove when there is a dpkg package built
+RUN pip3 install prometheus-flask-exporter
```

Comment:
OK, so few questions....

1. Why do we need this here in the e2e tests? Can we not just skip all prometheus here?
2. are we not adding Prometheus to any o the other docker configs? Meaning, are we no t adding it to the release docker configs? or do you want to wait and test a bit before doing so?

Reply:
This isn't needed. I will disable the ppa e2e tests until the packages are built

Reply:
I added a temp install from pip in the PPA GH action. 
Will remove as soon as the packages are built:
https://github.com/google/timesketch/issues/1656